test/features/keycloak.feature

```diff
@@ -111,7 +111,10 @@ Feature: Kogito integration with Keycloak
         "modelIdentifier": {
           "resourceType": "dmn",
           "resourceId": "namespace:name"
-        }
+	},
+	"type":"LIME",
+	"inputs": {},
+	"outputs": {}
```

Comment:
can you please be careful with the indentation ?

Reply:
woups this is GH display which is buggy ... sorry

Reply:
Actually I missed a space. Added